src/board/generic_esp32/generic-esp32-board.c

```diff
@@ -58,25 +143,21 @@ uint32_t BoardGetRandomSeed( void )
     return (mac_addr_bin[5] << 0) | (mac_addr_bin[4] << 8) | (mac_addr_bin[3] << 16) | (mac_addr_bin[2] << 24);
 }
 
+#include "nvs.h"
+
 void BoardGetUniqueId( uint8_t *id )
```

Comment:
are you sure you want to use the DEVEUI here? all other implementations use something hardware unique - see   

https://github.com/Lora-net/LoRaMac-node/blob/99c2e5378dde37e3331bf3efdf9df8f77d4847ee/src/boards/NucleoL152/board.c#L221

Reply:
Yes I'm glad you called this out, I was iffy on this commit. I am dropping it

Reply:
I'm not sure of the reason either. Searching online about it, it seems like a bad idea to leave the reset pin floating/doesn't make any difference. I think I am going to leave it as disabled for now for these reasons.